### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by drgomesp** [ibraimgm/libcmd#8]
Since last commit is from over 2 years ago, I guess that's a fair question to ask.

**Comment by ibraimgm** [comment]
Fair enough, I know for a fact that it is used in production in some of the companies that I've worked earlier (it was born from the needs of some of them). No commit was made in the last two years because it is already feature complete in the places where it is being used (but I reckon that I should at least release a "proper" version and update the README).

If you're having a problem with this library or need any help/guidance, feel free to ask.
